docs/DeveloperGuide.md

```diff
@@ -347,13 +347,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
-
-{More to be added}
+`* * *` | user | add events | help myself make tasks more manageable
+`* * *` | user | add tasks | help myself make tasks more manageable
+`* * *` | user | add deadlines | help myself make tasks more manageable
+`* * *` | user | CRUD events, tasks, deadlines | help myself make tasks more manageable
+`* * *` | user | undo last operation | help myself make tasks more manageable
+`* * *` | user | have some flexiblity in commands | help myself make tasks more manageable
+`* * *` | user | simple search for finding an item | help myself make tasks more manageable
+`* * *` | user | view uncompleted/completed tasks | help myself make tasks more manageable
```

Comment:
@okkhoy This was due to a miscommunication! Will sort it out (: Thanks for highlighting this!